CriticalMass/FriendsVerificationController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MapController.swift
```

Comment:
maybe rename here although this is redundant info anyways. But we love XCode of course :D

Reply:
I think that looks pretty good. Nice job

Reply:
Can not really say a lot about implementation. I will dig into it though

Reply:
Are you okay with merging it into the feature branch?